app/src/main/res/layout/login_fragment.xml

```diff
@@ -50,6 +50,18 @@
 
                 </com.google.android.material.textfield.TextInputLayout>
 
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="Here's a hint"/>
+
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/show_hint"
+                    android:textColor="@color/blue"
```

Comment:
see if secondary or secondary dark color is looking good. If that is looking good then replace it otherwise keep blue

Reply:
I have changed the color according to the theme
<img src="https://user-images.githubusercontent.com/70805716/123387615-9a552400-d5b5-11eb-9de2-96f876d86a7f.png" width="300">